libs/service-portal/family/src/screens/FamilyMember/FamilyMember.tsx

```diff
@@ -28,9 +28,7 @@ const NationalRegistryFamilyQuery = gql`
     nationalRegistryFamily {
       fullName
       nationalId
```

Comment:
Is this information not available in the new Soap service?

Reply:
Nope:
```
Had to take out address and maritalStatus from the familyMember since the new
client doesn't return that. Needs a further refactor in the next slice if we want it.
```

397b88d
"This needs to be split up into another component where a nationalRegistryUser is queried for the matching familyMember"